packages/dashboard/src/components/tasks/task-logs.tsx

```diff
@@ -1,33 +1,48 @@
 import { Divider, Grid, Paper, PaperProps, styled, Typography, useTheme } from '@mui/material';
-import { TaskEventLog } from 'api-client';
+import { TaskEventLog, TaskState, LogEntry } from 'api-client';
 import { format } from 'date-fns';
-import React from 'react';
-
 const prefix = 'task-logs';
 const classes = {
   root: `${prefix}-root`,
 };
 
 interface TaskLogProps {
   taskLog: TaskEventLog;
+  taskState: TaskState;
   fetchTaskLogs?: () => Promise<never[] | undefined>;
 }
 
 const StyledPaper = styled((props: PaperProps) => <Paper variant="outlined" {...props} />)(
   ({ theme }) => ({
     [`&.${classes.root}`]: {
-      padding: theme.spacing(2),
-      width: 350,
-      marginLeft: theme.spacing(2),
+      padding: theme.spacing(1),
+      width: '95%',
       flex: '0 0 auto',
+      maxHeight: '95%',
+      overflow: 'auto',
     },
   }),
 );
 
 export function TaskLogs(props: TaskLogProps) {
   const { taskLog } = props;
+  const { taskState } = props;
```

Comment:
```suggestion
const { taskLog, taskState } = props;
```

You can also do it directly on the function params

```ts
export function TaskLogs({ taskLog, taskState }: TaskLogProps) {
```